packages/bff/package.json

```diff
@@ -14,7 +14,7 @@
     "build": "tsc",
     "check": "tsc --project tsconfig.check.json",
     "generate-model": "pnpm generate-model-bff && pnpm generate-model-tenant-process && pnpm generate-model-catalog-process && pnpm generate-model-attribute-process && pnpm generate-model-agreement-process && pnpm generate-model-purpose-process && node --loader ts-node/esm ./patchZodios.ts",
-    "generate-model-bff": "mkdir -p ./src/model/generated && pnpm openapi-zod-client './open-api/bff-spec.yml' -o './src/model/generated/api.ts'",
+    "generate-model-bff": "mkdir -p ./src/model/generated && pnpm openapi-zod-client --group-strategy tag './open-api/bff-spec.yml' -o './src/model/generated/api.ts'",
```

Comment:
To be removed once this [PR](https://github.com/pagopa/interop-be-monorepo/pull/700) is merged